tests/integration/conftest.py

```diff
@@ -446,8 +480,50 @@ def azure_subscription_id(self):
 
 
 @pytest.fixture
-def az_cli_ctx(ws, env_or_skip):
-    return LocalAzureCliTest(ws, env_or_skip)
+def az_cli_ctx(ws, env_or_skip, make_schema, sql_backend, make_table, make_group, make_udf):
+    ctx = LocalAzureCliTest(ws, env_or_skip, make_schema, make_table, make_udf, make_group)
+    return ctx.replace(workspace_client=ws, sql_backend=sql_backend)
+
+
+class LocalAwsCliTest(TestRuntimeContext, WorkspaceContext):
```

Comment:
```suggestion
class LocalAwsCliTest(TestRuntimeContext):
```
this hints that `TestRuntimeContext` is not a correct name of the class. It could be renamed to `TestCliContext`.